proto/dymension/incentives/gauge.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+package dymensionxyz.dymension.incentives;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "osmosis/lockup/lock.proto";
```

Comment:
@danwt after the `proto-gen` refactor, what needed to be added to support this import? currently it fails